variables.tf

```diff
@@ -121,6 +121,12 @@ variable "protocol" {
   description = "The protocol for the created ALB target group (if `target_group_arn` is not set)"
 }
 
+variable "protocol_version" {
+  type        = string
+  default     = "HTTP1"
+  description = "The protocol_version for the created ALB target group (if `target_group_arn` is not set and if protocol is either HTTP or HTTPS)"
```

Comment:
Can we use the same description as the upstream provider docs

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#protocol_version